src/Config.php

```diff
@@ -1402,6 +1402,12 @@ function formatGlobalLayout(layout) {
             echo "</tr></table>";
 
             echo "</td></tr>";
+
+            echo "<tr class='tab_bg_2'>";
+            echo "<td><label for='dropdown_use_flat_dropdowntree_on_search_result$rand'>" . __('Display the tree dropdown complete name on search results') . "</label></td><td>";
+            Dropdown::showYesNo('use_flat_dropdowntree_on_search_result', $data["use_flat_dropdowntree_on_search_result"], -1, ['rand' => $rand]);
+            echo "</td>";
+            echo "</tr>";
```

Comment:
Should be moved in `showFormUserPrefs`, and be displayed next to `use_flat_dropdowntree` preference.